AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -3908,49 +3908,46 @@ function Debug-AzStorageAccountEntraKerbAuth {
             Write-Host "Checking Entra Object"
             try {
                 $checksExecuted += 1;
-                $Context = Get-AzContext
-                $TenantId = $Context.Tenant
-
+                $TenantId = $context.Tenant

```

Comment:
Suggestion: to avoid repeating `$TenantId = $context.Tenant` and `$accountUriObject.DnsSafeHost`, maybe extract these to variables defined at the top?

```powershell
$tenantId = $context.Tenant
$fileEndpoint = $accountUriObject.DnsSafeHost
```